umei-openapi.json

```diff
@@ -1672,6 +1672,17 @@
             "type": "string",
             "description": "A small description of the market",
             "nullable": true
+          },
+          "quantityType": {
+            "type": "string",
+            "enum": [
+              "Power",
```

Comment:
It would be better to specify here "ActivePower". Simply "Power" could be interpreted as total power which is a combination of active and reactive power.

Reply:
I agree with you Arnaud